atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraReloadableKvsConfig.java

```diff
@@ -237,7 +252,7 @@ public String type() {
 
     @Override
     public int concurrentGetRangesThreadPoolSize() {
-        return config.concurrentGetRangesThreadPoolSize();
+        return poolSize() * servers().numberOfThriftHosts();
```

Comment:
This is going to drop overrides of `concurrentGetRangesThreadPoolSize` in install configuration.